clients/cloud/python/producer.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+#
+# Copyright 2018 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# =============================================================================
+#
+# Produce messages to Confluent Cloud
+# Using Confluent Python Client for Apache Kafka
+#
+# =============================================================================
+
+from confluent_kafka import Producer, KafkaError
+from confluent_kafka.admin import AdminClient, NewTopic
+import sys
+import json
+
+
+if __name__ == '__main__':
+    if len(sys.argv) != 3:
```

Comment:
Maybe worth checking if Python has a nicer command argument parser (@rnpridgeon will know). Mostly because it will make the example code a bit more similar to what one should run in production.

Reply:
`argparse` isn't amazing but it is part of the standard library and its better than parsing the arguments yourself. 

https://github.com/confluentinc/confluent-kafka-python/blob/master/examples/avro-cli.py#L172-L189